whois-api/src/main/java/net/ripe/db/whois/api/rest/marshal/StreamingMarshalTextPlain.java

```diff
@@ -32,26 +45,42 @@ public <T> void writeArray(T t) {
     }
 
     @Override
-    public void close() {
+    public <T> void singleton(T t) {
         try {
+            outputStreamWriter.write(t.toString());
             outputStreamWriter.close();
         } catch (IOException e) {
             throw new StreamingException(e);
         }
     }
+
     @Override
-    public <T> void throwNotFoundError(HttpServletRequest request, List<Message> errorMessages) {
```

Comment:
I think that you don't need to override this method because you already have the default method with same code in the interface, right?

Reply:
same with createErrorStringEntity and createErrorStringMessages

Reply:
I pull the branch in my local and I see that those methods are different.